templates/telemetry_derived/main_summary_v4/part1.sql

```diff
@@ -0,0 +1,398 @@
+WITH sample AS (
```

Comment:
r? @akkomar on the main summary query itself

Reply:
Nice!
Did you compare output of this query to main_summary produced from BQ? Are they identical?

Reply:
they are definitely not identical yet, working on tracking down all the differences

Reply:
> Did you compare output of this query to main_summary produced from BQ? Are they identical?

they are now identical for `submission_date = '2019-10-27' AND sample_id = 42` with the one exception being some histograms and scalars are `NULL` slightly more often in dataproc than here. I suspect that is due to json path case sensitivity, but haven't confirmed it yet.

Reply:
I think this is acceptable.